source/fundamentals/typescript.txt

```diff
@@ -17,6 +17,10 @@ In this guide, you can learn about the **TypeScript** features and limitations
 of the MongoDB Node.js driver. TypeScript is a strongly typed programming
 language that compiles to JavaScript.
 
+The TypeScript compiler offers error and type checking in real time. Code
```

Comment:
S: The connection between "error" and "checking" gets lost because of the intervening words. Do you have specific errors in mind that the transpiler catches apart from those found via type checking? If not, I'd suggest just leaving out the error bit:

```suggestion
The TypeScript compiler offers type checking in real time. Code
```

Reply:
Good suggestion!